odbc-api/src/connection.rs

```diff
@@ -240,6 +245,26 @@ impl<'c> Connection<'c> {
     pub unsafe fn promote_to_send(self) -> force_send_sync::Send<Self> {
         force_send_sync::Send::new(self)
     }
+
+    /// Returns the maximum length of catalog names in bytes.
```

Comment:
Minor nitpick: I'd prefer 'Maximum length of catalog names in bytes' to 'Returns the maximum length of catalog names in bytes.'